### PR TITLE
fix: Action did not work on Windows runners.

### PR DESCRIPTION
**Pull request by sokuhatiku**
#### Changes
Fix action to work on `windows-latest`
- Replaced symlink to simple javascript.
- Fixed wrong UnityEditor paths.
- Fixed wrong project path at Standalone tests.
- Added `nographics` option to Editor command line variable.

Added tests to verify that action works on `windows-latest`
- Specified runs-on bash for replacing script backend  process for IL2CPP.

#### Related Issues

- #200

#### Successful Workflow Run Link

- https://github.com/sokuhatiku/unity-test-runner/actions/runs/7463004561

#### Notes

- This PR adds `-nographics` option to the Unity command line arguments on windows runner. This change affects tests that use graphics features such as RenderTexture for example.
-  To see if tests work correctly with windows-latest, this PR increases number of test patterns in main.yml from 49 to 79.

#### Checklist

- [x] Read the contribution [guide](../CONTRIBUTING.md) and accept the [code](../CODE_OF_CONDUCT.md) of conduct
- [ ] Docs (If new inputs or outputs have been added or changes to behavior that should be documented. Please make a PR
      in the [documentation repo](https://github.com/game-ci/documentation))
- [x] Readme (updated or not needed)
- [x] Tests (added, updated or not needed)

### CODE REVIEW

File: dist/main.js
Comment (GabLeRoux) [inline dist/main.js:1]: Building on macOS (unix), I had the following diff about file permissions:

```diff
diff --git a/dist/index.js b/dist/index.js
index 9d3ce2f..ec401f7 100644
Binary files a/dist/index.js and b/dist/index.js differ
diff --git a/dist/index.js.map b/dist/index.js.map
index e9255d8..45e40c0 100644
Binary files a/dist/index.js.map and b/dist/index.js.map differ
```

I don't know if this could have an impact. If we want this change, we can cherry-pick 32df7d8e805a9c77e4b68f45ed8cd29512073b0d which I pushed on `fix/windows-runner-dist-644` branch.

**Review by GabLeRoux — APPROVED** [review]
I added some small comments while reviewing. Overall, this looks good to me. I think it's worth having an additionnal review on this before merging and creating a release.

Thanks a lot for your contributions [sokuhatiku](https://github.com/sokuhatiku)! :D

File: dist/platforms/windows/run_tests.ps1
Comment (sokuhatiku) [reply dist/platforms/windows/run_tests.ps1:122]: I believe the code you suggested is cleaner. However, I followed the indentation rules used in [activate.ps1](https://github.com/game-ci/unity-test-runner/blob/ce07478/dist/platforms/windows/activate.ps1#L18-L28) and [return_license.ps1](https://github.com/game-ci/unity-test-runner/blob/ce07478/dist/platforms/windows/return_license.ps1#L16-L26) and wrote it the same way. Therefore, I leave it to you to decide which one to adopt.

**Review by davidmfinol — APPROVED** [review]
Looks good!

**Comment by webbertakken** [comment]
@sokuhatiku Would you mind rebasing and regenerating the dist files? Should be straight forward.

I think it would be great to merge this one.

**Comment by sokuhatiku** [comment]
@webbertakken I've rebased and regenerated the dist files as requested.

File: src/index.ts
Comment (webbertakken) [reply src/index.ts:2]: LGTM

**Review by webbertakken — APPROVED** [review]

**Pull request merged**
